### PR TITLE
Update to k8s 1.12.6 and pkg @ HEAD

### PR DESCRIPTION
**Pull request by mattmoor**
I had to pin a ton of packages to get things working.  In every case I basically just pinned the version in knative/serving.

### CODE REVIEW

**Comment by mattmoor** [comment]
Ran `./hack/update-codegen.sh` with the new code-generator

**Comment by mattmoor** [comment]
/hold

I may have screwed up the Status inlining.

**Comment by mattmoor** [comment]
Nevermind, it was the `Status` field in spec that I was seeing.
/hold cancel

File: Gopkg.toml [hunk 1]
```diff
@@ -19,36 +19,70 @@ required = [
...
+  revision = "511bab8e55decfbe784cd2d1868fa5cda62dc6c1"
+
+[[override]]
+  name = "github.com/census-instrumentation/opencensus-proto"
+  # What serving has in Gopkg.lock
+  revision = "7f2434bc10da710debe5c4315ed6d4df454b4024"
+
+[[override]]
+  name = "golang.org/x/oauth2"
+  # What serving has in Gopkg.lock
```
Comment (ImJasonH) [inline Gopkg.toml]: This comment will get out of date almost immediately. Is there a better way to describe why these overrides are here? Why do we need them?

Comment (mattmoor) [reply Gopkg.toml]: Now that these are locked, we can actually just remove these...  🙄

**Pull request merged**